projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -698,7 +703,7 @@ export class MapOlService {
     }
 
     if (l.bbox) {
-      layeroptions.extent = transformExtent(l.bbox, WGS84, this.map.getView().getProjection().getCode());
+      layeroptions.extent = transformExtent(l.bbox.slice(0, 4) as [number, number, number, number], WGS84, this.map.getView().getProjection().getCode());
```

Comment:
Maybe when working on #72 we can also try to refactor `map-ol.service.ts` so we use it's own functions `this.getProjection().getCode()`

Reply:
done!